server.elm

```diff
@@ -2033,72 +2270,310 @@ editionscript = """
+
+          When other elements in selector are clicked, change 'clicked element' to it. And it also follow rules above.
+        */
+        domSelectorDiv.classList.add("dom-selector-style");
+        domSelectorDiv.append(
+          el("div", {"class": "mainElem"}, []),
+          el("div", {"class": "childrenElem"}, [])
+        );
+
+        let displayMainElem = function(elem) {
+          let mainElemDiv = document.querySelector(".dom-selector > .mainElem");
+          mainElemDiv.append(
+            el("div", {"class":"mainElemName", "type":"text", value: elem.tagName.toLowerCase()}, "<" + elem.tagName.toLowerCase() + ">", {
+              onmouseenter: (c => () => { c.setAttribute("ghost-hovered", "true") })(elem),
+              onmouseleave: (c => () => { c.removeAttribute("ghost-hovered") })(elem)
+            }),
+            el("div", {"class": "mainElemInfo"}, textPreview(elem, 50)) // display its text content
+          )
+        }
+
+        let displayChildrenElem = function(elem) {
+          let childrenElemDiv = document.querySelector(".dom-selector > .childrenElem");
+          childrenElemDiv.append(
+            el("div", {"class": "childrenSelector"},
+              [
+                el("div", {"class": "childrenSelectorName"}, "<" + elem.tagName.toLowerCase() + ">", {}),
+                el("div", {"class": "childrenSelectorInfo"}, textPreview(elem, 20))
+              ], 
+              {
+                onmouseenter: (c => () => { c.setAttribute("ghost-hovered", "true") })(elem),
+                onmouseleave: (c => () => { c.removeAttribute("ghost-hovered") })(elem)
+              }
+            )
+          );
+        }
+
+        let displayElemAttr = function(targetDiv, elem) {
+          // show attribute of element on the dom selector
+          for (let i = 0; elem && elem.attributes && i < elem.attributes.length; i++) {
+            let name = elem.attributes[i].name;
+            let value = elem.attributes[i].value;
+            if (name === "ghost-clicked" || name === "ghost-hovered") continue;
+            targetDiv.append(
+              el("div", { "class": "elementAttr" },
+                [
+                  el("span", { title: "This element has attribute name '" + name + "'" }, name + ": "),
+                  el("span", { title: "This element has attribute value '" + value + "'" }, value)
+                ]
+              )
+            );
+          }
+        }
+
+        if (clickedElem.id !== "context-menu" || clickedElem.id !== "modify-menu" || clickedElem.id !== "editbox") {
+          if (selectorStatus === 1) {
+            // status 1
+            // display clicked element in main part
+            let mainElemDiv = document.querySelector(".dom-selector > .mainElem");
+            displayMainElem(clickedElem);
+            domSelectorDiv.classList.add("selectedDom");
+            mainElemDiv.addEventListener('click', (c => event => {
```

Comment:
Please set the propery `.onclick`. Adding an event listener is useful only in case if we shared the node with another library.